### DISCUSSION TITLE
Export as SVG

### DISCUSSION THREAD

**Discussion by btbritz** [visgl/deck.gl#6316] [Q&A]
I'm wondering if exporting the deckgl canvas as SVG is possible being that canvas elements are raster by default. I'm able to get a PNG and JPEG no problem using `toDataURL` but all of the libraries I've seen require that you first draw to the ctx of the board using native canvas API before serializing, which I believe deck.gl doesn't do? 

Long question short, is there a way to get an SVG of the canvas? If so, how?

Using the latest deckgl version and Scatterplotlayer and Textlayer if that makes a difference.

**Comment by Pessimistress** [comment]
> is there a way to get an SVG of the canvas?

No. WebGL and SVG run on completely different logics.
